### DISCUSSION TITLE
Disable auto entry on Enter

### DISCUSSION THREAD

**Discussion by mramendi** [pulsar-edit/.github#206] [Core Application & Packages]
So I am editing AsciiDoc and Pulsar does autocompletion on the Enter key, whcih I use to create a newline. And sometimes it's really overzealous. Most notably, if I type : and then press Enter, it adds "name: value:", apparently identifying the line as an "attribute entry". So the common "Enter this command:" results in a weird line unless I notice this in time.

How can I disable this autocompoletion on Enter for such specific cases? Or if that is not possible how do I remove autocompletion on Enter entirely, moving it to a different key/combination in case I really need to autocomplete something?

**Comment by Spiker985** [comment]
I do believe that the `autocomplete-plus` package is handling that.

There are a few options in its package settings to tweak how it suggests entries, and how to select entries - perhaps you can try there?

**Reply by Spiker985** [reply]
You can also, at any time, use the `keybinding-resolver` (Ctrl + .) to see what each key combination is mapped to, and override that behavior in your `keymaps` config

**Comment by savetheclocktower** [comment]
Go to Settings, then Packages, then find `autocomplete-plus` and click on <kbd>Settings</kbd>.

The setting you want to change is called **Keymap For Confirming A Suggestion**. I suggest changing it to “tab always, enter when suggestion explicitly suggested.” That will stop it from treating <kbd>Enter</kbd> as choosing an autocompletion suggestion in the scenario you describe.

You could also increase the value in **Delay Before Suggestions Are Shown** so that the menu doesn't appear quite as quickly.
